cookiecutter.json

```diff
@@ -0,0 +1,12 @@
+{
+  "maintainer_name": "Audrey Roy Greenfeld",
+  "maintainer_email": "audrey@deltares.nl",
+  "project_name": "New Chess",
```

Comment:
"New Chess"  was my work-title due to a lack of imagination. Probably  should replace this with  "Design Toolkit Application" ?

Reply:
Done